doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -0,0 +1,128 @@
+.. _third_party_x_compilers:
+
+3rd Party Toolchains
+####################
+
+A "3rd party toolchain" is an officially supported toolchain provided by an
+external organization. Several of these are available.
+
+GNU ARM Embedded
+****************
+
+#. Download and install a `GNU ARM Embedded`_ build for your operating system
+   and extract it on your file system.
+
+   .. warning::
+
+      Like the Zephyr repository, do not install the toolchain in a directory
+      with spaces anywhere in the path.
+
+#. Configure the environment variables needed to inform the Zephyr build system
+   to use this toolchain:
+
+   - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``gnuarmemb``.
+   - Set :envvar:`GNUARMEMB_TOOLCHAIN_PATH` to the toolchain installation
+     directory.
+
+   For example, if you installed to :file:`<user folder>/gnu_arm_embedded` (or
```

Comment:
To avoid issues with spacing (many users have spaces in their username) I recommend that instead you use `c:\gnu_arm_embedded` as an example path for Windows.